keyboards/splitkb/kyria/rev2/keymaps/vial/rules.mk

```
@@ -0,0 +1,38 @@
+# MCU name
+MCU = atmega32u4
+
+# Bootloader selection
+BOOTLOADER = caterina
+
+# Build Options
+#   change yes to no to disable
+#
+BOOTMAGIC_ENABLE = no      # Enable Bootmagic Lite
+MOUSEKEY_ENABLE = yes 		# Mouse keys
+EXTRAKEY_ENABLE = yes      # Audio control and System control
+SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
```

If you're really concerned about file size, leaving mousekeys disabled frees up around 1800 bytes.

```suggestion
MOUSEKEY_ENABLE = yes 		# Mouse keys
```
`BOOTMAGIC_ENABLE` is forcibly set to yes if `VIA_ENABLE` is set to `yes`, so attempting to set it to `no` here does nothing: https://github.com/vial-kb/vial-qmk/blob/4eb1c2e21fef3b2a94215992158c0195b92a8d4d/builddefs/common_features.mk#L609-L615

`EXTRAKEY_ENABLE = yes` is already set at the keyboard level: https://github.com/vial-kb/vial-qmk/blob/4eb1c2e21fef3b2a94215992158c0195b92a8d4d/keyboards/splitkb/kyria/rules.mk#L12

`SLEEP_LED_ENABLE` is disabled by default, so attempting to set it to `no` here does nothing: https://github.com/vial-kb/vial-qmk/blob/4eb1c2e21fef3b2a94215992158c0195b92a8d4d/builddefs/common_features.mk#L530-L535

---

good to know, thanks. I was desperate to try various options to reduce the firmware size